Formula/caffe.rb

```diff
@@ -0,0 +1,53 @@
+class Caffe < Formula
+  desc "Fast open framework for deep learning"
+  homepage "http://caffe.berkeleyvision.org/"
+  url "https://github.com/BVLC/caffe/archive/1.0.tar.gz"
+  sha256 "71d3c9eb8a183150f965a465824d01fe82826c22505f7aa314f700ace03fa77f"
+
+  head "https://github.com/BVLC/caffe.git"
```

Comment:
please remove `head`

Reply:
May I ask why? (And why does the audit tool not mention it?)

Reply:
We occasionally add `head` if a user requests it for their own purposes but not as a matter of course.